app.js

```diff
@@ -9,3 +9,114 @@ document.querySelectorAll('.menu-link').forEach((k) => k.addEventListener('click
   menu.classList.remove('active');
   windowMenu.classList.remove('active');
 }));
+
+// work section content
+/* eslint-disable no-restricted-globals */
+
+const cardArray = [
+  {
+    title: 'facebook',
+    img: 'Portfolio1.png',
```

Comment:
- [ ] Currently, from my end the images of some of the card on the main page as well as on the pop-up modal is not showing (i.e the link is broken). Kindly target the right directory of the image and be sure they are displaying perfectly 👍 

design 👇 

![ahlo2](https://user-images.githubusercontent.com/78634841/216146824-50143759-c10d-4b8f-8936-40096829810d.png)

Reply:
hello, I'm really confused about the images because on my end they are all loading up : 
![image](https://user-images.githubusercontent.com/92299875/216151860-4fd8fbb1-1e62-4b87-b7b4-8b10dca9c52c.png)